src/config/rcxml.c

```diff
@@ -516,7 +590,18 @@ entry(xmlNode *node, char *nodename, char *content)
 		fill_usable_area_override(nodename, content);
 	}
 	if (in_keybind) {
-		fill_keybind(nodename, content);
+		if (in_action_query) {
+			fill_action_query(nodename, content,
+				current_keybind_action);
```

Comment:
Could we have `if (in_action_{query,then_branch,else_branch})` at the same level as `in_{key,mouse}bind` if we use of a global (within translation unit) `parent_action` variable?

Reply:
I'm not sure I'm following.

Reply:
Ah, I get it. In order to get rid of code duplication between mousebind and keybind?

Reply:
Sounds good, but let's do it afterwards.

Reply:
Yes, exactly. Fine with one step at a t ime.